examples/csharp/HelloworldUWP/GreeterServer/GreeterServer.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Protobuf Include="Protos\greet.proto" GrpcServices="Server" Generator="MSBuild:Compile" />
+    <Content Include="@(Protobuf)" />
+    <None Remove="@(Protobuf)" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Grpc.AspNetCore.Server" Version="0.1.20-pre1" />
```

Comment:
Grpc.AspNetCore.Server lives in a different repository https://github.com/grpc/grpc-dotnet, so pehaps it doesn't make sense to showcase it in examples in this repository. 
Also, because the server is compatible with the GreeterServer from the basic example, perhaps it makes more sense to only provide a UWP GreeterClient example and provide instructions in rREADME.md to run it against the greeterServer from Helloworld example?

Reply:
OK, will do.